src/components/layout/RegisterNameForm/styles.css

```diff
@@ -0,0 +1,47 @@
+.register-name-modal {
+  display: flex;
+  flex-direction: column;
+  max-width: 1000px;
+  min-width: 320px;
+  width: 80%;
+  height: fit-content;
+  margin: auto;
+  padding: 2em 2em;
+  gap: 2em;
+  background: var(--card-bg);
+  border: 1px solid #323232;
+  border-radius: 32px;
```

Comment:
i personally don't care for the shaded box around the workflow, but know that's what figma portrays - let's see what the feedback is at the next demo for this